### ISSUE TITLE
Configuring optimizer arguments

### ISSUE THREAD

**Issue by alfrem** [maikherbig/AIDeveloper#7]
Arguments for each optimizer are different and would be nice to be able to be configured as seen in:

https://keras.io/optimizers/

Learning rate is already in place, would be perfect to have Rho, beta_1, beta_2, amsgrad, momentum and nesterov.

Great job, by the way.

**Comment by maikherbig** [comment]
Hi,
I thought about that and actually included them in an initial design of the UI (see below). But then I decided to make it more "simple" as the default parameters are typically just fine. Do you like the design below? Do you change these other parameters often?
![MoreOptimizer_Parameters](https://user-images.githubusercontent.com/14074051/79476857-cd8f4100-8009-11ea-8373-0de5a31c7e61.png)

**Comment by alfrem** [comment]
Seems perfect and very functional to me :). You are great!

**Comment by maikherbig** [comment]
Solved via  c7689b1
I decided to make the menu less apparent by adding just a little button (...) next to the Optimizer Drop-down menu. The button will invoke a popup-menu looking similar as shown in my comment above.
Sorry it took me so long. I focused working on implementing learning rate schedules (cyclical learning rates etc). In my experience, the learning rate is one of the more important hyper parameters.
